pkg/machine/qemu/config.go

```diff
@@ -42,6 +54,75 @@ type MachineVM struct {
 	UID int
 }
 
+type MachineVM struct {
+	// The command line representation of the qemu command
+	CmdLine []string
+	// HostUser contains info about host user
+	HostUser
+	// ImageConfig describes the bootable image
+	ImageConfig
+	// Mounts is the list of remote filesystems to mount
+	Mounts []Mount
+	// Name of VM
+	Name string
+	// PidFilePath is the where the PID file lives
+	PidFilePath MachineFile
+	// QMPMonitor is the qemu monitor object for sending commands
+	QMPMonitor Monitor
+	// ReadySocket tells host when vm is booted
+	ReadySocket MachineFile
+	// ResourceConfig is physical attrs of the VM
+	ResourceConfig
+	// SSHConfig for accessing the remote vm
+	SSHConfig
+}
+
+// ImageConfig describes the bootable image for the VM
+type ImageConfig struct {
+	IgnitionFilePath string
+	// ImageStream is the update stream for the image
+	ImageStream string
+	// ImagePath is the fq path to
+	ImagePath string
+	// Memory in megabytes assigned to the vm
```

Comment:
Memory missing?

Reply:
IMHO I think comment `// Memory in megabytes assigned to the vm` should be removed from here and `memory` should be only part of `type ResourceConfig` and not of `type ImageConfig struct`.